python/interpret_community/widget/js/main.js

```diff
@@ -1,23 +1,40 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
 
 import { ExplanationDashboard } from 'interpret-dashboard';
 
 const RenderDashboard = (divId, data) => {
   let generatePrediction = (postData) => {
-    return fetch(data.predictionUrl, {method: "post", body: JSON.stringify(postData), headers: {
-      'Content-Type': 'application/json'
-    }}).then(resp => {
-      if (resp.status >= 200 && resp.status < 300) {
-        return resp.json()
-      }
-      return Promise.reject(new Error(resp.statusText))
-    }).then(json => {
-      if (json.error !== undefined) {
-        throw new Error(json.error)
-      }
-      return Promise.resolve(json.data)
-    })
+    if (data.withCredentials) {
```

Comment:
does the axios path not handle both credintialed and non-credentialed requests? I think localhost would just ignore the headers it didn't need. Would be nice to have a single code-path if possible

Reply:
it doesn't handle the non-credentialed request unless I remove withCredentials parameter, but then I think it's better to use the previous fetch code which is simpler and is known to work instead